apps/server/test/lexical/server/code_intelligence/completion/translations/macro_test.exs

```diff
@@ -612,4 +612,29 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Translations.MacroTest do
       assert completion.sort_text == "dunder_macro/0"
     end
   end
+
+  test "test has three completions", %{project: project} do
+    # Error:
+    # right: {:error, :not_found}
+    assert {:ok, completion} =
+             project
+             |> complete("test|")
+             |> fetch_completion("test ")
```

Comment:
so I was expecting this to work fine, but I suppose I need to have ExUnit in the context of `project`. I need to explore the code for `project` more to figure out how to do this.

If you already know how to do this easily, please tell me :D

Reply:
When you use the `complete` function, you're passing in source code, and all the context that it has. The entirety of your source code here is `test|`, and as `test` isn't something imported from `Kernel`, you have no completions. Instead, give the completer more context, a you would in an actual test

```elixir
code = ~q[
defmodule FakeTest do 
  use ExUnit.case 
  te|
]

assert {:ok, completion} = 
  project
  |> complete(code)
  |> fetch_completion("test ")
```